### ISSUE TITLE
Fehler in SCSS killt Redaxo-Backend

### ISSUE THREAD

**Issue by alexplusde** [Sioweb/redaxo_assets#20]
Hatte (vermutlich durch die falsche Reihenfolge von SCSS-Dateien ein "undefined mixin"-Fehler, woraufhin das Redaxo-Backend nicht mehr lädt und ich die fehlerhafte Einstellung in Assets auch nicht mehr rückgängig machen konnte.

Wäre es nicht besser, SCSS nur im Frontend zu parsen?

**Comment by Sioweb** [comment]
Nein, im Backend gibt es ebenfalls den Bedarf CSS einzusetzen, um das Look&Feel den Company-Standards anzupassen. Zudem ermöglicht es einigen Addons mit Sass/Less zu arbeiten.

Wenn du mir Zugang zu einem Test-Projekt gibst, kann ich es mir gerne mal anschauen.

**Comment by alexplusde** [comment]
Sorry, dass ich mich erst so spät melde. Im Prinzip kannst du das SCSS-Plugin kicken, weil Redaxo von Haus aus ja schon einen mitbringt.

Wird gerade hier diskutiert:
https://github.com/FriendsOfREDAXO/scss_compiler/issues/4#issuecomment-232363198
